brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java

```diff
@@ -27,8 +28,14 @@
   private final MongoCollection<MongoVerdict> collection;
   private final PublishSubject<String> subject = PublishSubject.create();
 
-  public MongoTransactionVerdictStorage() {
-    mongoClient = MongoClients.create();
+  public MongoTransactionVerdictStorage(String mongoHost, Integer mongoPort) {
+    if (Strings.isNullOrEmpty(mongoHost)) {
+      throw new IllegalArgumentException("MongoDB host must not be neither null or empty");
+    }
+    if (mongoPort < 1) {
```

Comment:
Do we really need this check? If so, it looks like we also need to add an upper bound check

Reply:
Done